gpu_direct_rdma_access.h

```diff
@@ -61,6 +61,22 @@ struct rdma_open_dev_attr {
     int             ib_port;
     int             gidx;
 };
+
+enum rdma_task_attr_flags {
+        RDMA_TASK_ATTR_RDMA_READ = 1 << 0,
+        RDMA_TASK_ATTR_ZERO_BYTE_MSG = 1 << 1
```

Comment:
this should be expose in user API.
you're using it internally ro have better code reuse of the submit_task()... need some other way to do it.